cpp/map_module/algorithm/map.cpp

```diff
@@ -1 +1,86 @@
 #include "map.hpp"
+#include <string>
+#include "mg_utils.hpp"
+#include "mgp.hpp"
+
+namespace {
+std::string value_to_string(const mgp::Value &value) {
```

Comment:
This function supports key being any type that isn't null, list, map or path. There wasn't a simple way to implement it (except null which can be ignored) but I don't think it's very important at the moment.
This function could perhaps be added inside mgp::Value